lib/repo/git_hooks/server/post-receive.d/01-show_push_time.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+# Do not show time if no change have been made to the master branch
+cat - | cut -d' ' -f 3 | grep -qw 'refs/heads/master' || exit 0
+
+echo "[MARKUS] Your submission has been received: $(git reflog -n 1 --date=local --format=format:%gd master)"
```

Comment:
Similar to above, please modify `[MARKUS]` to `[MarkUs]` (and in the other hooks below)